docs/alesco/meeting-minutes/2024-08-21.md

```diff
@@ -0,0 +1,80 @@
+# ALESCo Meeting Minutes (2024-07-24)
+Minutes recorded by Jonathan Wright.
+
+Edited by Jonathan Wright for publishing.
+
+## Members
+### ALESCo Member Attendees
+- Jonathan Wright
+- Cody Robertson
+- Andrew Lukoshko
+- Elkhan Mammadli
+- Ben Thomas
+- Neal Gompa
+  
+### Board Attendees
+- benny Vasquez
+- Alex Iribarren
+
+## Community Attendees
+
+## Decisions Adopted
+1. SPICE support to be added back to AlmaLinux 10
+   - Removed in RHEL 9
+   - Will revisit for enablement in AlmaLinux 9 at a later time
+   - Unanimous decision
+
+## Minutes
+- SPICE Support in AlmaLinux
+  - Neal: Dropped in RHEL due to no RHEL stacks dependent on it.
```

Comment:
Mind adding the rhbz reference I pulled for this decision too? So people can see full context.

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=2030592#c4

Reply:
Still got that link handy?

Reply:
^

Reply:
Note that SPICE supports VP9 (which addresses the issue about H264): https://gitlab.freedesktop.org/spice/spice/-/blob/master/server/gstreamer-encoder.c#L845-862